core/shared/src/main/scala/cats/effect/IO.scala

```diff
@@ -944,7 +944,6 @@ sealed abstract class IO[+A] private () extends IOPlatform[A] {
         if (NonFatal(t) && runtime.config.reportUnhandledFiberErrors) {
           runtime.compute.reportFailure(t)
         } else {
```

Comment:
I thought this was Daniel Urban's point about this :)

```suggestion
        if (NonFatal(t)) {
          if (runtime.config.reportUnhandledFiberErrors)
            runtime.compute.reportFailure(t)
        } else {
```

Reply:
ohhhhhh 🤦🏻‍♀️ 🤦🏻‍♀️ 🤦🏻‍♀️